app/components/Views/confirmations/Approval/index.js

```diff
@@ -469,14 +469,14 @@ class Approval extends PureComponent {
 
       // For Ledger Accounts we handover the signing to the confirmation flow
       if (isLedgerAccount) {
-        const ledgerKeyring = await getLedgerKeyring();
+        const deviceId = getDeviceId();
```

Comment:
Missing `await` here:

```suggestion
        const deviceId = await getDeviceId();
```

This should fix the signing issue with Ledger